.github/workflows/maven-release.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow will put the project in our staging repo
+name: Releasing Project to maven
```

Comment:
Looks good to me. Is there some way to test this?

Reply:
https://github.com/aws-cloudformation/aws-cloudformation-resource-schema/runs/1683532098?check_suite_focus=true